code/code.py

```diff
@@ -9,6 +9,8 @@
 setting_private_variable_formatter   = mod.setting('code_private_variable_formatter', str)
 setting_protected_variable_formatter = mod.setting('code_protected_variable_formatter', str)
 setting_public_variable_formatter    = mod.setting('code_public_variable_formatter', str)
+setting_type_formatter               = mod.setting('code_type_formatter', str)
```

Comment:
I'm not very familiar with typescript, but isn't this covered by the _*_variable_formatters?

Reply:
Oops, nevermind. Derp.